issue/338-modals-extend-past-fold

mtlewando opened the pull request:
Minor CSS tweaks to ensure content scrolls within modal and modal doesn't extend beyond the fold. Also disable main site scroll when modal is open.

Modal header not included in scroll to ensure close button remains in place.

Will resolve #338

moshfeu commented on src/components/Modal/Modal.css:
```diff
@@ -26,12 +29,15 @@
   background: #fff;
   max-width: 780px;
   width: 95%;
+  /* height: 80vh; */
```
```suggestion

```

moshfeu commented on src/components/Modal/Modal.css:
```diff
@@ -26,12 +29,15 @@
   max-width: 780px;
   width: 95%;
+  /* height: 80vh; */
+  /* max-height: 80vh; */
```
```suggestion

```

moshfeu commented on src/components/Modal/Modal.css:
```diff
@@ -3,14 +3,17 @@
   left: 0px;
   pointer-events: none;
-  overflow: scroll;
+  overflow: none;
```
`overflow: none;` it's not valid property.

```suggestion

```

mtlewando replied:
Doh! Not sure where this came from. 🙄

mtlewando commented:
Thanks @moshfeu. Apologies for not cleaning my comments out before making a PR!

For the body, it's possibly overkill but I've switched the CSS to:
```css
.is-modal {
  overflow-y: scroll;
  position: fixed;
}
```
This will stop the body content scrolling but keeps the vertical scrollbar so prevents the cards jumping. Probably the most lightweight solution I could think of.

moshfeu commented:
This is a valid solution too. Although, I think that we will stay with the way the [landing page](https://codingcoach.io/) behave - not preventing the scroll.
